website/content/docs/agent-and-proxy/agent/versions.mdx

```diff
@@ -2,21 +2,30 @@
 layout: docs
 page_title: Vault Agent Version Compatibility
 description: |-
-  Guidelines for running different versions of Agent and Server
+  Guidelines for running different versions of Agent and Vault server
```

Comment:
"Vault Server"

Reply:
It looks like elsewhere (as per the suggestions Sarah made) we're normalized on "Vault server", so I'll keep it as-is for consistency